README.md

```diff
@@ -6,7 +6,10 @@ This typically allows accessing servers via
 `service.local.dev-gutools.co.uk`, rather than a `localhost:PORT` URL,
 which among other things makes it possible to share cookies for the [pan-domain authentication](https://github.com/guardian/pan-domain-authentication).
 
+Unfamiliar with nginx? [More details here](./EXPLAINER.md).
```

Comment:
Why a separate doc?

Reply:
Up to you! I can move it to README if preferred.

Reply:
I think we should have this in the main README as it's super helpful.